apis/Google.Cloud.Video.LiveStream.V1/.OwlBot.yaml

```diff
@@ -0,0 +1,31 @@
+
+# Copyright 2021 Google LLC
```

Comment:
nit: This one is still beeing generated with 2021

Reply:
Ooh, interesting. Thanks, will fix.

Reply:
(In a separate PR though, as I suspect it's the generator.)